pkg/console/starter/starter.go

```diff
@@ -445,10 +445,12 @@ func RunOperator(ctx context.Context, controllerContext *controllercmd.Controlle
 			// "DefaultIngressCertValidation",
 			// in 4.13 we are removing CustomRouteSync and DefaultRouteSync
 			// we are need to backport the removal of these conditions all the way down to 4.10
-			// since we only remove the in https://github.com/openshift/console-operator/pull/662
-			// and its cause upgrade issues to the customers
-			"CustomRouteSync",
-			"DefaultRouteSync",
+			// since we only remove them in https://github.com/openshift/console-operator/pull/662
+			// and its causing upgrade issues to the customers
+			"CustomRouteSyncDegraded",
+			"CustomRouteSyncProgressing",
+			"DefaultRouteSyncDegraded",
+			"DefaultRouteSyncProgressing",
```

Comment:
Not sure if you want to respond to [this feedback][1] before you land this 4.13 backport or not.

[1]: https://github.com/openshift/console-operator/pull/757#discussion_r1176511988

Reply:
@wking not sure what you mean

Reply:
Oops, sorry for the delayed response.  @yanpzhan was [asking][1] if you wanted to include `DefaultRouteSyncUpgradeable` and `CustomRouteSyncUpgradeable` in the conditions you clear.  Checking Telemetry with:

```
sort_desc(count by (condition, reason) (cluster_operator_conditions{name="console",condition="Upgradeable",reason=~".*RouteSync.*"}))
```

turns up some reasons like `ConsoleCustomRouteSync_FailedDeleteCustomRoutes` and `ConsoleDefaultRouteSync_FailedDefaultRouteApply`, so I expect you do want to delete those too?

[1]: https://github.com/openshift/console-operator/pull/757#discussion_r1176511988

Reply:
No we donw need to remove these, since they are the proper ones